src/main/java/de/tum/in/www1/artemis/web/websocket/dto/metis/CrudAction.java

```diff
@@ -0,0 +1,9 @@
+package de.tum.in.www1.artemis.web.websocket.dto.metis;
+
+/**
+ * Enum that is used in DTOs sent as payload in websocket messages,
+ * it is used to differentiate the trigger for a websocket message and define subsequent actions in the client components
+ */
+public enum CrudAction {
```

Comment:
Just from looking at this class, it would not be clear to me what this enum is used for. I suggest renaming the class to something like `MetisCrudAction` and to be more specific on the comment.